docs/root/operations/admin.rst

```diff
@@ -190,7 +190,48 @@ The fields are:
 
   .. http:get:: /stats?format=json
 
-  Outputs /stats in JSON format. This can be used for programmatic access of stats.
+  Outputs /stats in JSON format. This can be used for programmatic access of stats. Counters and Gauges
+  will be in the form of a set of (name,value) pairs. Histograms will be under the element "histograms",
+  that contains "supported_quantiles" which lists the quantiles supported and an array of computed_quantiles
+  that has the computed quantile for each histogram.
+
+  The JSON structure would as shown below.
+
+  .. code-block:: json
+
+  {
+    "stats": [
+        {
+            "name": "example.counter",
+            "value": 0
+        },
+        {
+            "name": "example.gauge",
+            "value": 0
+        },
+        {
+            "histograms": {
+                "supported_quantiles": [...],
```

Comment:
Could you add real values here, so the user knows what the array will look like (as in, is it an array of `doubles` or subobjects)?

Reply:
will add.